style.css

```diff
@@ -1 +1,245 @@
 /*  Add your styling here */
+@import url('https://fonts.googleapis.com/css2?family=Rubik:wght@300;400&display=swap');
+
+
+
+:root {
+    --light-pink: 
+}
```

Comment:
Very nicely done! Using css variables for your color definitely helps with readability.  :)